content/library/api-cheat-sheet.md

```diff
@@ -117,6 +117,28 @@ st.video(data)
 
 <CodeTile>
 
+#### Display charts
```

Comment:
Last Wednesday when browsing through the docs, I was puzzled why we had never included this. Thank you for adding the charting commands to the cheat sheet!